Frameworks/UIKit/UITextField.mm

```diff
@@ -323,20 +306,33 @@ - (void)setTextColor:(UIColor*)color {
  @Status Interoperable
 */
 - (UIColor*)textColor {
-    return _textColor;
+    WUXMSolidColorBrush* colorBrush = nil;
+    if (_secureTextMode) {
+        colorBrush = rt_dynamic_cast<WUXMSolidColorBrush>(_passwordBox.background);
+        if (colorBrush != nil) {
+            return ConvertWUColorToUIColor(colorBrush.color);
+        } else {
+            return nil;
+        }
+    } else {
+        colorBrush = rt_dynamic_cast<WUXMSolidColorBrush>(_textBox.background);
```

Comment:
yes ;/

---
In reply to: [95470285](https://github.com/Microsoft/WinObjC/pull/1604#discussion_r95470285) [](ancestors = 95470285)